Add precisions about DEV plan on custom extensions (#280)
CleverCloud/documentation: content/doc/addons/postgresql/_index.md

pgvector    | Vector data type and ivfflat and hnsw access methods
timescaledb | Enables scalable inserts and complex queries for time-series data (Community Edition)

{{< callout type="warning" >}}

> On-demand extensions are not available for DEV addons.

Reviewer: ```suggestion
On-demand extensions are not available for DEV plans.
```

Reviewer: Also, maybe use "aren't" like the bot said? :shrug:

Reviewer 2: Yeah, we unified the term. It's `add-on` and it flags `addon`

```suggestion
On-demand extensions aren't available for DEV plans.
```
